typings/index.d.ts

```diff
@@ -2665,6 +2665,17 @@ declare module 'hypixel-api-reborn' {
       blaze: SKYBLOCK_SLAYER_DATA;
       vampire: SKYBLOCK_SLAYER_DATA;
     };
+    kuudra: {
```

Comment:
Types will also need changing for a return type

Reply:
i added an `| null` after the kuudra type. Should be fixed?

Reply:
ye